protocol/simplex-messaging.md

```diff
@@ -348,16 +351,18 @@ Each transmission between the client and the server must have this format/syntax
 
 ```abnf
 transmission = [signature] SP signed SP pad ; pad to the fixed block size
-signed = [corrId] SP [queueId] SP cmd
+signed = sessionIdentifier SP [corrId] SP [queueId] SP cmd ; corrId is required in client commands and server responses,
```

Comment:
Define `sessionIdentifier` here? It is way below

Reply:
or add "See `sessionIdentifier` in [Appendix A - SMP transport protocol](#appendix-a)

Reply:
yes, the latter

Reply:
it's actually defined above